src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/LocateMatchingApplicantsController.java

```diff
@@ -0,0 +1,70 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.exception.FigaroValidationServiceException;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.LocateMatchingApplicantsRequest;
+import ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice.types.LocateMatchingApplicantsResponse;
+
+/**
+ * 
+ * LocateMatchingApplicantsController  class. 
+ * 
+ * @author shaunmillargov
+ *
+ */
+@RestController
+public class LocateMatchingApplicantsController {
+	
+	@Autowired
+	private FigaroValidationImpl figservice;  // connection to ORDS client. 
+	
+	@RequestMapping(value = "/locateMatchingApplicants",
+	  produces = { "application/xml" }, 
+	  method = RequestMethod.GET)
+	public LocateMatchingApplicantsResponse locateMatchingApplicants(
+			@RequestParam(value="applSurname", defaultValue="") String applSurname,
+			@RequestParam(value="applFirstName", defaultValue="") String applFirstName,
+			@RequestParam(value="applSecondInitial", defaultValue="") String applSecondInitial,
+			@RequestParam(value="applBirthDate", defaultValue="") String applBirthDate,
+			@RequestParam(value="applDriversLicence", defaultValue="") String applDriversLicence,
+			@RequestParam(value="applBirthPlace", defaultValue="") String applBirthPlace,
+			@RequestParam(value="applGenderTxt", defaultValue="") String applGenderTxt,
+			@RequestParam(value="applAliasSurname1", defaultValue="") String applAliasSurname1,
+			@RequestParam(value="applAliasFirstName1", defaultValue="") String applAliasFirstName1,
+			@RequestParam(value="applAliasSecondInitial1", defaultValue="") String applAliasSecondInitial1,
+			@RequestParam(value="applAliasSurname2", defaultValue="") String applAliasSurname2,
+			@RequestParam(value="applAliasFirstName2", defaultValue="") String applAliasFirstName2,
+			@RequestParam(value="applAliasSecondInitial2", defaultValue="") String applAliasSecondInitial2,
+			@RequestParam(value="applAliasSurname3", defaultValue="") String applAliasSurname3,
+			@RequestParam(value="applAliasFirstName3", defaultValue="") String applAliasFirstName3,
+			@RequestParam(value="applAliasSecondInitial3", defaultValue="") String applAliasSecondInitial3) throws FigaroValidationServiceException {
```

Comment:
Should the controller throws FigaroValidationException ?

Reply:
No. Any FigaroValidationException raised by code within the controller body should be captured with it's failure message added to the response type (LocateMatchingApplicantsResponse) and sent back to the called. This will maintain the HTTP status code of 200 the legacy system expects but indicate what the error was to the caller (KOFAX). I'll add this code once we start into wiring up the ORDS client calls.